_includes/maps/state-areas.html

```diff
@@ -0,0 +1,21 @@
+    <g class="states features">
+      <use xlink:href="{{ include.svg }}#states"></use>
+    </g>
+    {% assign keys = include.value %}
+    {% for state in include.states %}
+    {% assign _state = site.states | where: 'id', state[0] | first %}
+    {% if include.href %}<a xlink:href="{{ include.href | format_url: _state }}">{% endif %}
```

Comment:
The link to individual states isn't working on Federalist. I think we need to add a `{{ site.baseurl }}` to the href here